### PR TITLE
Approvers and issuers events condense

### PR DESCRIPTION
**Pull request by hshar7**
- Combined approvers events
- Combined issuers events
- This way we get all the indexes each time. Better data integrity.

### CODE REVIEW

File: contracts/StandardBounties.sol [hunk 1]
```diff
@@ -174,7 +174,7 @@ contract StandardBounties {
   /// @param _issuers the array of addresses who will be the issuers of the bounty
   /// @param _approvers the array of addresses who will be the approvers of the bounty
   /// @param _data the IPFS hash representing the JSON object storing the details of the bounty (see docs for schema details)
-  /// @param _deadline the timestamp which will become the deadline of the bounty
+  /// @param _deadline the time0stamp which will become the deadline of the bounty
```
Comment (c-o-l-o-r) [inline contracts/StandardBounties.sol]: extra 0 here

File: contracts/StandardBounties.sol [hunk 2]
```diff
@@ -644,10 +640,11 @@ contract StandardBounties {
   validateIssuerArrayIndex(_bountyId, _issuerId)
   onlyIssuer(_sender, _bountyId, _issuerId)
   {
-    for (uint i = 0; i < _approvers.length; i++){
-      bounties[_bountyId].approvers.push(_approvers[i]);
-    }
-    emit BountyApproversAdded(_bountyId, _sender, _approvers);
+  for (uint i = 0; i < _approvers.length; i++){
+  bounties[_bountyId].approvers.push(_approvers[i]);
```
Comment (c-o-l-o-r) [inline contracts/StandardBounties.sol]: need indent

File: contracts/StandardBounties.sol [hunk 3]
```diff
@@ -509,13 +509,12 @@ contract StandardBounties {
     public
     senderIsValid(_sender)
     validateBountyArrayIndex(_bountyId)
-    validateIssuerArrayIndex(_bountyId, _issuerId)
```
Comment (mbeylin) [inline contracts/StandardBounties.sol]: Why did you remove this line? This validates the array index of the sender of the message, which is different from the index of the issuer who needs to be changed.

Comment (hshar7) [reply contracts/StandardBounties.sol]: Stack too deep issue started to happen once I made the changes of the emitted values. I thought this one can be compromised since we are checking for it indirectly using onlyIssuer()

Comment (c-o-l-o-r) [reply contracts/StandardBounties.sol]: We should figure out why this happened, because `validateIssuerArrayIndex` is used in many places in the contract.

Comment (hshar7) [reply contracts/StandardBounties.sol]: It's because https://ethereum.stackexchange.com/questions/6061/error-while-compiling-stack-too-deep
